### DISCUSSION TITLE
Prevent user from simultaneous logins to multiple thinclients

### DISCUSSION THREAD

**Discussion by knightflyer** [ltsp/ltsp#478] [Q&A]
I cannot find a way to prevent this in the client section.  It causes problems, such as in chrome I get a dialog reporting another host has the same profile open, do you want to close the other profile, and then chrome does not run. The simplest solution in my humble opinion would have an option preventing this.

Thanks for a great useful package.

**Comment by alkisg** [comment]
Hi, see #357
